### ISSUE TITLE
Warn when navigating away from unsaved editors

### ISSUE THREAD

**Issue by Throne3d** [Marri/glowfic#170]
> It would be nice if clicking the "add alias" button while editing a character did at least one of the following:
> - automatically saved changes to the character
> - asked if you wanted to save changes to the character
> - alerted you that you would lose unsaved changes to the character
> - gave you a pop-up / new text entry field, instead of loading a new page
>
> I keep editing a character, putting new things in several text fields, and getting down to the "add alias" button and hitting that and then realizing that it loads a new page and grumbling at my brain.

-[Moriwen](http://alicorn.elcenia.com/board/viewtopic.php?f=12&t=459&start=1970#p32458)

(More generally this should probably be added to a variety of forms, but only appear when the content has been modified from default.)

**Comment by Marri** [comment]
Follow up with a link to character pages from the editor, since accidental clicks won't lose anything.

**Comment by Marri** [comment]
@Throne3d didn't you add this?

**Comment by Throne3d** [comment]
I don't think so – I did do *something* like warning when moving away from editors, iirc, but not this. (I can't recall what I *did* do though…)
